### DISCUSSION TITLE
Repository Example

### DISCUSSION THREAD

**Discussion by 201flaviosilva** [Andrew-Chen-Wang/github-wiki-action#13] [Q&A] [answered]
Hey there,

Nice work, but there is any repository example working? 😅

**Comment by Andrew-Chen-Wang** [comment, marked as answer]
Sure, take a look at some users here: https://github.com/Andrew-Chen-Wang/github-wiki-action/network/dependents?package_id=UGFja2FnZS0yOTQ2Mjc5Njcy

**Reply by 201flaviosilva** [reply]
Hi, thank you very much for your quick response, but this is not what I wanted to ask. 🤦
My question is here: https://github.com/Andrew-Chen-Wang/github-wiki-action/discussions/14

Sorry man, again 🤦
